docs/tac-skills.md

```diff
@@ -157,7 +157,6 @@ cd tac_controller_contract
 ``` bash
 aea add connection fetchai/oef:0.2.0
 aea add skill fetchai/tac_control_contract:0.1.0
-aea add contract fetchai/erc1155:0.2.0
```

Comment:
this is not needed anymore. Contracts are correctly added if a skill depends on them (see https://github.com/fetchai/agents-aea/issues/1045), and the skill `fetchai/tac_control_contract:0.1.0` depends on `fetchai/erc1155:0.2.0`.

Reply:
Yes it wasn't previously the case but now it is you are correct. Should be changed for the participants as well.